src/components/ShareTrajectoryButton/index.tsx

```diff
@@ -24,31 +26,41 @@ const ShareTrajectoryButton = ({
         setIsSharing(!isSharing);
     };
 
-    const handleTooltipOpen = () => {
-        if (isSharing || !simulariumFile.name) return false;
+    const isDisabled = () => {
+        return !simulariumFile.name || isSharing || isBuffering;
+    };
+
+    const getTooltipOffset = () => {
+        if (isDisabled()) {
+            return [0, -30];
+        } else return [0, -18];
```

Comment:
again, maybe it's just personal preference, but I feel like just having boolean constants here would nice

```
const isDisabled = !simulariumFile.name || isSharing || isBuffering;
const tooltipOffset = isDisabled ? [0, -30] : [0, -18];
```